functions/nodejs-apps.bash

```diff
@@ -1,27 +1,42 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2181
 
+## Function for installing NodeJS for frontail and other addons.
+##
+##    nodejs_setup()
+##
 nodejs_setup() {
-  if [ -x "$(command -v npm)" ]; then return 0; fi
+  if [[ -x $(command -v npm) ]]; then return 0; fi
 
+  local link
   local myDistro
+  local temp
 
-  if ! [ -x "$(command -v lsb_release)" ]; then
+  if ! [[ -x $(command -v lsb_release) ]]; then
     echo -n "$(timestamp) [openHABian] Installing NodeJS prerequsites (lsb-release)... "
     if cond_redirect apt-get install --yes lsb-release; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
 
+  link="https://unofficial-builds.nodejs.org/download/release/v12.18.2/node-v12.18.2-linux-armv6l.tar.xz"
```

Comment:
use a link to last if possible

Reply:
Unfortunately it is not so I will just keep an eye on it.